src/Containers/Game/Game.scss

```diff
@@ -48,4 +48,71 @@ h5 {
 
 .option-button {
   background: $btn-white;
+}
+
+
+
+.hint-button {
+  background-color: $btn-gray;
+  box-shadow: 0 3px 8px rgba(0, 0, 0, .3);
+  color: $blue-accent;
+  margin: 1.25rem auto;
+  padding: 12px 0;
+  text-align: center;
+  text-shadow: 0 0 1rem black;
+  width: 40%;
+}
+
+.hint-button-disabled {
+  box-shadow: 0 2px 5px rgba(0, 0, 0, .1);
+  color: #11111D;
+  text-shadow: none;
+}
+
+.option-button {
+  background: $btn-white;
+}
+
+@media screen and (min-width: 535px) {
+  .Game {
+    max-width: 960px;
+    margin: 0 auto;
+    overflow: hidden;
+  }
+
+  .account-area {
+    background: $btn-gray;
+  }
+
+  .flag-main {
+    height: unset;
+    margin: 5rem auto 0;
+    max-height: 450px;
+    max-width: 600px;
+    width: unset;
+  }
+  
+  .flag-image {
+    max-height: 425px;
+    max-width: unset;
+    width: 100%;
+  }
+
```

Comment:
On this media query (min-width: 535px), the flag seems a bit large so that it causes the user to have to scroll down to see all 4 answer options.
I'd like to propose the following flag image size adjustment so that users can see all 4 answer options without having to scroll:

@media screen and (min-width: 535px)
.flag-image {
    max-height: 300px;
    max-width: unset;
    width: 80%;
}